05/Textarea.js

```diff
@@ -1,54 +1,38 @@
 import React from 'react';
 class Textarea extends React.Component {
 
-    /*tu chyba dość namieszałam... działa bo dziala, ale nie jestem z tego kodu zadowolona, nie wiem dlaczego resize zwraca false, powinno chyba true ? bo currentHeight jest większy od scroll więc dlaczego zwraca to co jest po lewej stronie czyli false ?. Prosze o wskazówki, poprawie :)  */
-
     extraHeight = 10
     maxHeight = 100
     
     textAreaRef = React.createRef();
 
-    handleText = event => {
-       const  {changeTextArea} = this.props;
-       const textAreaVaue = this.textAreaRef.current.value;
-
-       const currentHeight = this.textAreaRef.current.offsetHeight;
-       console.log(`currentHeight: ${currentHeight}`)
-       const scrollHeight = event.target.scrollHeight;
-       console.log(`scrollHeight: ${scrollHeight}`)
-
-
-       changeTextArea(textAreaVaue)
+    handleText = () => {
+        const  {changeTextArea} = this.props;
+        const textAreaVaue = this.textAreaRef.current.value;
 
+        changeTextArea(textAreaVaue)
     }
 
-    getSnapshotBeforeUpdate = (currentHeight, scrollHeight) => {
-      
-        return currentHeight > scrollHeight && currentHeight >= this.maxHeight 
-        ?
+    getSnapshotBeforeUpdate(prevProps, prevState) {
+        /*czemu to nie działa , źle zapisane? */
+    //    const shouldResize = this.textAreaRef.current.offsetHeight > this.textAreaRef.current.scrollHeight &&  this.textAreaRef.current.offsetHeight <= 100 ? true: false
 
-        {
-            resize: true
-        }
-        :
-        {
-            resize: false
+        return {
+            shouldResize:  (this.textAreaRef.current.offsetHeight > this.textAreaRef.current.scrollHeight &&  this.textAreaRef.current.offsetHeight <= 100 ) ?
```

Comment:
znak powinien być odwrotny tj. `(this.textAreaRef.current.offsetHeight > this.textAreaRef.current.scrollHeight &&  this.textAreaRef.current.offsetHeight <= 100 )`